packages/notification-service/test/firebase.test.ts

```diff
@@ -33,8 +33,17 @@ const mockedKnownAddressesCache = {
   },
 }
 
+const mockedRegistrationsRef = {
+  on: jest.fn((event, callback) =>
+    // @ts-ignore: Only mocking the `val` property
```

Comment:
Looking for suggestions on how to do this test better - now that we are not holding the db in memory anymore, i think we can mock either the database itself or the database reference (are there more options?). It is hard to mock these manually because these objects have many properties that we don't need. I saw this type of `ts-ignore` in this file already so i took a shortcut and contributed to the problem here. Is there a better way to mock the registrations? should i spend some time to find a lib that helps us with testing firebase-admin?